source/includes/administration/_notifications.md

```diff
@@ -0,0 +1,374 @@
+        }
+    ],
+    "category": {
+        "id": "2569f2cc-f1d7-4664-8439-173fedfab1c3"
+    }
+}
+```
+Create a notification.
+
+Required | &nbsp;
+---------- | -----------
+`category`<br/>*[Notification Category](#administration-notification-categories)]* | The notification category that the notification belongs to.<br/>*includes*: `id`
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of the translation.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating (must be one of: title, body, description, url slug).
+
+Optional | &nbsp;
+---------- | -----------
+`publishedAt`<br/>*string* | The date on which the notification has been/will be published.
+`deprecatedAt`<br/>*string* | The date on which the notification has been/will be deprecated.
+`splash`<br/>*boolean* | When set to true, the notification will be displayed in the home screen until it reaches its deprecated date. Only one of splash and sticky can be set to true.
+`sticky`<br/>*boolean* | When set to true, the notification will be displayed upon logging in. Only one of splash and sticky can be set to true.
+
+<!-------------------- UPDATE NOTIFICATION -------------------->
+
+#### Update notification
+
+`PUT /content/notifications/:id`
+
+```shell
+# Update notification
+curl -X PUT "https://cloudmc_endpoint/v2/rest/content/notifications/:id" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request-body"
+```
+> Request body example:
+
+```json
+{
+    "id":"e5412e1d-64ac-40b2-a1a1-05780584fc71",
+    "translations": [
+        {
+            "language": "en",
+            "text": "system-notification-create-no-autho",
+            "type": "title"
+        },
+        {
+            "language": "fr",
+            "text": "en français",
+            "type": "title"
+        },
+        {
+            "language": "en",
+            "text": "system notification create",
+            "type": "body"
+        },
+        {
+            "language": "fr",
+            "text": "en français",
+            "type": "body"
+        }
+    ],
+    "category": {
+        "id": "dc921a2a-16c2-4bc3-8ed6-bcb571bae241"
+    }
+}
+```
+Update a notification.
+
+Required | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification.
+`category`<br/>*[Notification Category](#administration-notification-categories)]* | The notification category that the notification belongs to.<br/>*includes*: `id`
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of the translation.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating (must be one of: title, body, description, url slug).
```

Comment:
This should only be title and body

Reply:
fixed 65d3185551293faa443a5e59617d81963fb934f1